.github/workflows/_build_test_upload.yml

```diff
@@ -52,153 +52,91 @@ jobs:
           - 3.9
+          - "3.10"
     steps:
       - name: Setup Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-      - name: Install Pre-release/Nightly-release/Official-release PyTorch
-        run: |
-          pip3 install numpy
-          # Add version requirement to PyTorch except nightly release
-          if [[ ${{ inputs.pytorch_version }} ]]; then
-            PYTORCH_VERSION=torch==${{ inputs.pytorch_version }}
-          else
-            PYTORCH_VERSION=torch
-          fi
-
-          PIP_CHANNEL=${{ needs.get_release_type.outputs.type }}
-
-          if [[ $PIP_CHANNEL == 'official' ]]; then
-            pip3 install "$PYTORCH_VERSION"
-          else
-            pip3 install --pre "$PYTORCH_VERSION" -f "https://download.pytorch.org/whl/$PIP_CHANNEL/cpu/torch_$PIP_CHANNEL.html"
-          fi
       - name: Checkout Source Repository
         uses: actions/checkout@v2
         with:
           ref: ${{ inputs.branch }}
-      - name: Get Hash of the Last Commit from Release
-        if: |
-          needs.get_release_type.outputs.type == 'nightly' ||
-          needs.get_release_type.outputs.type == 'test'
-        run: |
-          pip3 install -r requirements.txt
-          # Using --no-deps here to make sure correct version of PyTorch Core
-          pip3 install --pre torchdata -f "https://download.pytorch.org/whl/${{ needs.get_release_type.outputs.type }}/cpu/torch_${{ needs.get_release_type.outputs.type }}.html" --no-deps
-          pushd ~
-          RELEASE_COMMIT=$(python -c 'import torchdata; print(torchdata.version.git_version)')
-          echo "::set-output name=hash::$RELEASE_COMMIT"
-          popd
-          pip3 uninstall -y torchdata
-        id: release_commit
-      - name: Get Hash of the Last Commit from Source
-        if: |
-          needs.get_release_type.outputs.type == 'nightly' ||
-          needs.get_release_type.outputs.type == 'test'
-        run: |
-          SOURCE_COMMIT=$(git rev-parse HEAD)
-          echo "::set-output name=hash::$SOURCE_COMMIT"
-        id: source_commit
-      - name: Check if Build New TorchData Wheel
-        run: |
-          if [[ ${{ needs.get_release_type.outputs.type }} == 'official' || ( ${{ steps.release_commit.outputs.hash }} != ${{ steps.source_commit.outputs.hash }} ) ]]; then
-            echo 1 > torchdata_${{ matrix.os }}_${{ matrix.python-version }}_whl.txt
-            echo "::set-output name=value::true"
-          else
-            echo 0 > torchdata_${{ matrix.os }}_${{ matrix.python-version }}_whl.txt
-            echo "::set-output name=value::false"
-          fi
-        id: trigger_build
-      # The following steps for pre-release will be skipped if the last commit
-      # from branch is the same as the last commit in release wheel
-      - name: Build and Install TorchData Wheel
-        if: steps.trigger_build.outputs.value == 'true'
```

Comment:
Removing this part to check if we need to create wheel if there is no change in TorchData because I figured out PyTorch core may have a new nightly release even though TorchData doesn't have any change. In order to provide the up-to-date nightly release for both PyTorch and TorchData, this part of workflow should be removed.